common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -174,8 +205,8 @@ public Random current() {
                 DIRECT_MEMORY_COUNTER = new AtomicLong();
             }
         }
-        logger.debug("-Dio.netty.maxDirectMemory: {} bytes", maxDirectMemory);
-        DIRECT_MEMORY_LIMIT = maxDirectMemory >= 1 ? maxDirectMemory : MAX_DIRECT_MEMORY;
+        DIRECT_MEMORY_LIMIT = maxDirectMemory;
+        logger.debug("-D{}: {} bytes", JAVA_SYS_PROP_IO_NETTY_MAX_DIRECT_MEMORY, maxDirectMemory);
```

Comment:
revert this is not a change we did https://github.com/netty/netty/compare/netty-4.1.27.Final...sirensolutions:siren-4.1.27-5